src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -0,0 +1,38 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import javax.persistence.*;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity(name = "order")
+public class Order {
+    enum OrderStatus {
+        UNPAID,
+        PAID,
+        CANCELED
+    }
+
+    @Id
```

Comment:
uu mnie bez takiej adnotacji nie dało sie dodać rekordów do tabeli w Workbenchu bez ręcznego wpisywaniania ID, krzyczał że brakuje w zapytaniu ID, a sql powinien sam je generować

@GeneratedValue(strategy = GenerationType.IDENTITY)

Reply:
ok, dodałem na wszelki wypadek, ale nie widzę zeby coś to zmieniło

Reply:
przyznaję że  twojego brancha nie testowałem w workbenchu i może ten problem co u mnie nie występował, ale w sumie też fajnie będzie mieć ID we wszystkich encjach tak samo opisane